main.tf

```diff
@@ -407,8 +421,94 @@ resource "kubernetes_namespace" "kpi_namespace" {
   }
 }
 
+resource "kubernetes_namespace" "community_namespace" {
+  metadata {
+    name = "community"
+  }
+}
+
 resource "kubernetes_namespace" "ingress_nginx_namespace" {
   metadata {
     name = "ingress-nginx"
   }
 }
+
+resource "helm_release" "rocket-chat_deployment" {
+  name      = "rocket-chat"
+  chart     = "stable/rocketchat"
+  namespace = kubernetes_namespace.community_namespace.metadata.0.name
+  set {
+    name  = "image.tag"
+    value = "2.2.0"
+  }
+
+  set {
+    name  = "mongodb.mongodbPassword"
+    value = "putMeInCredentials"
+  }
+
+  set {
+    name  = "mongodb.mongodbRootPassword"
+    value = "putMeInCredentials"
+  }
+
+  set {
+    name  = "mongodb.enabled"
+    value = "true"
+  }
+
+  #   set {
+  #     name  = "ingress.enabled"
+  #     value = "true"
+  #   }
+
+  # ingress.annotations	Annotations for the ingress	{}
+  # ingress.path	Path of the ingress	/
+  # ingress.tls
+}
+
+data "helm_repository" "ory" {
+  name = "ory"
+  url  = "https://k8s.ory.sh/helm/charts"
+}
```

Comment:
I suppose that you still need to include the hydra module here?

Reply:
oh, I forgot to push my last commit^^